index.html

```diff
@@ -101,5 +101,129 @@ <h1>
         </svg>            
     </a>
+    <section id="portfolio">
+        <h1>My Recent<br>Works
+            <div id="underline"></div>
+        </h1>
+        <div class="portfolio-item">
+            <img src="" alt="">
+            <div class="portfolio-main">
+                <h2>Multi-Post Stories<br>Gain+Glory</h2>
+                <ul>
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+        </div>
+        <div class="portfolio-item">
+            <img src="" alt="">
+            <div class="portfolio-main">
+                <h2>Multi-Post Stories<br>Gain+Glory</h2>
+                <ul>
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+        </div>
+        <div class="portfolio-item">
+            <img src="" alt="">
+            <div class="portfolio-main">
+                <h2>Multi-Post Stories<br>Gain+Glory</h2>
+                <ul>
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+        </div>
+        <div class="portfolio-item">
+            <img src="" alt="">
+            <div class="portfolio-main">
+                <h2>Multi-Post Stories<br>Gain+Glory</h2>
+                <ul>
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+        </div>
+        <div class="portfolio-item">
+            <img src="" alt="">
+            <div class="portfolio-main">
+                <h2>Multi-Post Stories<br>Gain+Glory</h2>
+                <ul>
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+        </div>
+        <div class="portfolio-item">
+            <img src="" alt="">
+            <div class="portfolio-main">
+                <h2>Multi-Post Stories<br>Gain+Glory</h2>
+                <ul>
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+        </div>
+    </section>
+    <section id="about">
+        <div id="about-heading">
+            <h1>About me</h1>
```

Comment:
- [ ] Please consider replacing the h1 by an h2 or h3 tag as the page should have only one h1.